### ISSUE TITLE
Latest 3.6.6 does not works Denoiser option in Viewport rendering

### ISSUE THREAD

**Issue by odil24** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#647]
Sytem specs:
GPU - GTX 1060 6GB 
CPU - i3-10100F @ 3.6 ghz
RAM - 32 GB
OS - Win10 x64

Step to reproduce:
1. Just open up Blender
2. Select RPR engine 
3. Started viewporting rendering with Denoiser option ticked
4. Nothing denoised after finish

**Comment by odil24** [comment]
Is there have any reason for disabled?

**Comment by ARozghon** [comment]
yes, because image filter development for the "Final" render mode is outdated and now RPR used Open Image Denoiser for the Production render only. In the Interactive render used FSR+denoiser, but it not supported for the "Final" render mode

**Comment by odil24** [comment]
But how to activete that Interactive viewport denoising?

**Comment by odil24** [comment]
> yes, because image filter development for the "Final" render mode is outdated and now RPR used Open Image Denoiser for the Production render only. In the Interactive render used FSR+denoiser, but it not supported for the "Final" render mode

any responses?

**Comment by ARozghon** [comment]
if interactive mode does not appear in the list that's means GPU not supported by Interactive mode. and yes, Pascal GPUs are not supported

**Comment by odil24** [comment]
> if interactive mode does not appear in the list that's means GPU not supported by Interactive mode. and yes, Pascal GPUs are not supported 

Thanks for clarifications. It so sad for that but not problematic I think ;)
